### PR TITLE
svg_loader: fix clip transformation

### PR DESCRIPTION
**Pull request by mgrudzinska**
Bug introduced while refactoring 8492a63.

@Issue: https://github.com/thorvg/thorvg/issues/3083

### CODE REVIEW

**Review by JSUYA — APPROVED** [review]
LGTM

File: src/loaders/svg/tvgSvgSceneBuilder.cpp [hunk 1]
```diff
@@ -434,7 +433,7 @@ static bool _appendClipShape(SvgLoaderData& loaderData, SvgNode* node, Shape* sh
         }
     }
 
-    _applyProperty(loaderData, node, shape, vBox, svgPath, true);
+    _applyComposition(loaderData, shape, node, vBox, svgPath);
```
Comment (JSUYA) [inline src/loaders/svg/tvgSvgSceneBuilder.cpp:436]: This change does not affect the transformation of clips.
It will be reviewed again along with #3082.

**Review by hermet — CHANGES_REQUESTED** [review]
as @JSUYA comment.

**Comment by hermet** [comment]
https://github.com/thorvg/thorvg/pull/3079

**Pull request closed without merging**